python/pyspark/ml/classification.py

```diff
@@ -454,7 +454,8 @@ class TreeClassifierParams(object):
     impurity = Param(Params._dummy(), "impurity",
                      "Criterion used for information gain calculation (case-insensitive). " +
                      "Supported options: " +
-                     ", ".join(supportedImpurities), typeConverter=TypeConverters.toString)
+                     ", ".join(supportedImpurities) + ". (Default: gini)",
```

Comment:
@holdenk could we leave this change out of this PR and keep it only for links? When we make a final call on approach for #12914 we can revisit.